### ISSUE TITLE
utf8 characters in a string appear different in the editor vs serial terminal

### ISSUE THREAD

**Issue by rhubarbdog** [microsoft/pxt-microbit#2186]
**Describe the bug**
I thought i'd try unicode characters.  What is appearing in the makecode editor is different to what is output to a serail terminal (and over radio to a MicroPython program)

**To Reproduce**
i execute block `serial.writeString("96.3 ģĤĥ")` on press of a button
I'm running serial terminal `screen /dev/ttyACM0` and that displays `96.3 #$%`

**Expected behavior**
I expect `96.3 ģĤĥ` to be displayed in the serial terminal 

**Desktop (please complete the following information):**
 - OS: Linux mint 18.2
 - Browser mozilla firefox
 - Version 67

**Comment by abchatra** [comment]
Unicode support is not present in the hardware due to restricted character set. This is limitation of device due to low 16k RAM and 256 flash. Closing as Can't fix

**Comment by rhubarbdog** [comment]
I was following poor instructions as to typing utf8 charqcters under linux and the instructions were for any unicode chaacter.
I have printed the ° symbol from a serial terminal where i ptinted it as a MicroPython string which does use utf8 characters and copied and pasted it to the make code editor.  So I am expecting the degrees symbol to still be a utf8 character.

I have the following blocks
```
    if (input.buttonIsPressed(Button.B)) {
        string = "96.3" + "°" + "F"
        serial.writeString(string)
    }
```

but my serial terminal has the following when i press button B
96.3�F

do you support utf8 characters?
I have managed using this MicroPython module https://github.com/rhubarbdog/microbit-radio to send from MicroPython over radio to MakeCode blocks and echoed back the text "21.2°C" to MicroPython and  both the Blocks serial terminal and the MicroPython serial terminal correctly printed "21.2°C" on all occasions.

Have you converted strings to utf8 as per "UTF8 character support in strings" on web page https://support.microbit.org/support/solutions/articles/19000098698

**Comment by martinwork** [comment]
The micro:bit only deals with single byte characters. It's font is here: https://github.com/lancaster-university/microbit-dal/blob/master/source/core/MicroBitFont.cpp, which covers 95 ASCII characters, I think.

I searched online for "unicode degree symbol" and copy-pasted it into MakeCode. When sent via serial, this showed the symbol in the MakeCode console. In tera-term it showed a dot pattern until I switched the font.

degree is Unicode 00B0, so presumably gets transmitted as hex B0 which is also the degree symbol in Window ANSI code page 1252. I don't know how MakeCode deals with mapping characters to and from single byte.

The micro:bit happily sends the character B0 via serial, but "show string" does not display it on the LEDs.  

Receiving anything other than ASCII in a terminal program is going to depend on the host OS, language, font character set and the configuration of the terminal program.
